manticore/__main__.py

```diff
@@ -54,6 +54,8 @@ def positive(value):
     parser.add_argument('--workspace', type=str, default=None,
                         help=("A folder name for temporaries and results."
                               "(default mcore_?????)"))
+    parser.add_argument('--pure-symbolic', action='store_true',
+                        help='Treat all writable memory as symbolic')
```

Comment:
Should rather be moved to `bin_flags` and we might add a `(linux only)` as it won't work with decree targets.

Reply:
Good call; I originally thought evm stuff used `Memory` as well, but it doesn't. Will move.